plugins/modules/rds_cluster.py

```diff
@@ -25,9 +25,10 @@
   # General module options
     state:
         description: Whether the snapshot should exist or not.
-        choices: ['present', 'absent']
+        choices: ['present', 'absent', 'started', 'stopped']
         default: 'present'
         type: str
+        version_added: 6.3.0
```

Comment:
```suggestion
        
```

Reply:
Not sure why this was resolved. The `version_added` field still needs to be removed here.